### DISCUSSION TITLE
Score Visibility Improvements

### DISCUSSION THREAD

**Discussion by ikzune** [ppy/osu#20800] [Ideas] [answered]
When posting scores to somewhere like twitter which allow 4 screenshots at once, it is really hard to see individual screenshots without clicking on them to enlarge. The current score screen should have a simplified version that enlarges the song name and maybe the acc/combo/pp values which would be the main things people look at aside from the score which is large enough. prior to users clicking on the screenshot to enlarge. The below was captured on a 100% google chrome tab on a 1920x1080 monitor:
![Capture](https://user-images.githubusercontent.com/31851536/196201949-ddd209c1-63bf-4b97-bc8e-9aa04ea8fe1d.PNG)

I'm also not sure if the maximum text allocation is enough There also seems to be a low maximum text allocation in order to fit the space:
![osu_2022-10-16_23-10-55](https://user-images.githubusercontent.com/31851536/196202386-d6a3ebaa-4ae1-4bef-bfaf-befaebdc2aa3.jpg)

Also, it would be nice if we could modify the DT icon to indicate the speed increment being used on the score screen:
![osu_2022-10-17_20-37-41](https://user-images.githubusercontent.com/31851536/196205571-90e4a54a-21c8-4995-ae79-ef3e1ced587e.jpg)

Finally, is it worthwhile to have ranked status on the score screen?

**Comment by peppy** [comment, marked as answer]
The idea was to have a more detailed version display when clicking (ie. where it brings up the rest of the details). I don't know if any design work has been done on this yet (cc @arflyte ) but as noted in the [critical milestone task](https://github.com/ppy/osu/issues/18877) the current ranking screen is still missing a lot of information.

I agree the current screens are very shoddy to share. Which is like, the number one purpose of that screen.
